pkg/directorysync/client.go

```diff
@@ -99,10 +99,19 @@ type User struct {
 	// The identifier for the Organization in which the Directory resides.
 	OrganizationID string `json:"organization_id"`
 
+	// The User's primary email
+	Email string `json:"email"`
+
 	// The User's username.
+	// Deprecated: Will be removed in a future major version. Enable the `username` custom attribute
```

Comment:
Do you know if editors surface this up to the developers when using these attributes?

I see that [this is the convention](https://go.dev/wiki/Deprecated). I'm curious if this works with attributes in a struct.

Reply:
According to the doc you linked, this should show up as deprecated in editors, but in my editor all I see is the comment, no strikethrough or yellow squiggles or anything else.